src/bthread/task_meta.h

```diff
@@ -43,10 +48,10 @@ struct ButexWaiter;
 struct LocalStorage {
     KeyTable* keytable;
     void* assigned_data;
-    void* rpcz_parent_span;
+    std::weak_ptr<brpc::Span> rpcz_parent_span;
```

Comment:
I think you 'd better not use brpc types in the bthread code.
In the brpc design, bthread is a basic library, and brpc is built on top of it. So bthread should no depend on brpc. This is why the original type is `void*`, not `brpc::Span*`.
My suggestion: you can keep `void* rpcz_parent_span` here, and in the brpc code, you can new a `std::weak_ptr<brpc::Span>`, then cast the pointer `std::weak_ptr<brpc::Span>*` to `void*`. This may be more complicated, but keep the brpc design clean.